src/teleop/gui/src/components/Cache.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+<div>
+    <h3>Cache Controls</h3>
+    <button @pointerdown="velocity = -1*velocityScaleDecimal" @pointerup="velocity = 0" @mouseout="velocity = 0">Reverse</button>
+    <button @pointerdown="velocity = velocityScaleDecimal" @pointerup="velocity = 0" @mouseout="velocity = 0">Forward</button>
+    <input id="myRange" v-model="velocityScale" type="range" min="0" max="100">Velocity Scaling: {{velocityScale}}%</input>
+
+    <!-- Cache is open loop for now 11/9/22 -->
+    <!-- TODO: Add Indicator for cache being opened or closed once we have closed loop -->
```

Comment:
Has this changed since November? ICD still says "Maybe closed loop"

Reply:
The cache will no longer be supporting closed loop. Should I remove this comment?

Reply:
Yeah looks like we can remove and update line 12 to sound more permanent